src/api/spec/models/project_spec.rb

```diff
@@ -408,20 +409,67 @@
   describe '#self.valid_name?' do
     context "invalid" do
       it{ expect(Project.valid_name?(10)).to be(false) }
-      it{ expect(Project.valid_name?('home:M0ses:raspi::qtdesktop')).to be(false) }
+      it "has ::" do
+        property_of {
+          s1 = sized(1){ string(/[a-zA-Z0-9]/) }
+          s2 = sized(range(1, 199)){ string(/[-+\w\.]/) }
+          s = s1 + s2
+          i = range(0, (s.length - 2))
```

Comment:
It has `::` in any position of the string. So I want a string with a random size which also has `::` in a random position.

The RegEx is with the possible characters which could be in the string https://github.com/abargnesi/rantly/blob/master/lib/rantly/generator.rb#L229